make/photon/prepare/utils/configs.py

```diff
@@ -79,10 +78,9 @@ def validate(conf: dict, **kwargs):
         raise Exception(
             "Error: redis_host in harbor.yml needs to point to an endpoint of Redis server or cluster.")
 
-    redis_port = conf.get("redis_port")
-    if redis_host is None or (redis_port < 1 or redis_port > 65535):
+    if ':' not in redis_host:
```

Comment:
Is this checking the `{schema}://{host}`?

Reply:
no, this is check host:port in harbor.yaml
schema is a seperat field